ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -1865,3 +1868,19 @@ enum pkcs11_rc add_missing_attribute_id(struct obj_attrs **pub_head,
 	else
 		return set_attribute(pub_head, PKCS11_CKA_ID, id2, id2_size);
 }
+
+enum pkcs11_rc get_key_data_to_wrap(struct pkcs11_object *obj, void **data,
```

Comment:
Please move this right after `set_key_data()`

Reply:
Ok